spec/features/jobs/user_visits_jobs_sees_tags_spec.rb

```diff
@@ -29,9 +29,17 @@
 
   end
   scenario "user sees average salary for all jobs from spepcific tag listed" do
-    company = Company.create!(name: "ESPN")
-    job = company.jobs.create!(title: "Developer", level_of_interest: 70, city: "Denver")
 
+    company1 = Company.create!(name: "ESPN")
+    job1 = company1.jobs.create!(title: "Developer", level_of_interest: 70, city: "Denver", salary: 100000)
+    tag1 = job1.tags.create(name: "Software")
+    company2 = Company.create!(name: "WacDonalds")
+    job2 = company1.jobs.create!(title: "Developer"sinc, level_of_interest: 70, city: "FoCo", salary: 80000)
+    tag2 = job1.tags.create(name: "Software")
+
+
+    visit company_job_path(company1, job1)
 
+    expect(page).to have_content("Tags: #{tag1.name}, 2 (90000)")
   end
```

Comment:
Would love to see your expectations just a little more fleshed out here...can we expect to see the second tag name?